psamm/commands/fastgapfill.py

```diff
@@ -96,7 +96,11 @@ def reaction_key(r):
             penalties=penalties)
 
         epsilon = self._args.epsilon
-        core = set(self._mm.reactions)
+        coreall = set(self._mm.reactions)
+        core = set()
+        for r in coreall:
```

Comment:
I don't think it is necessary to save the `coreall` in a temporary set. How about just `for r in self._mm.reactions:`?